README.md

```diff
@@ -105,8 +105,20 @@ flag to specify the version. For example:
 bin/logstash-plugin install --version 6.4.2 logstash-output-amazon_es
 ```
 
-
-
+Starting in 7.2.0, the aws sdk version is bumped to v3. In order for all other AWS plugins to work together, please remove pre-installed plugins and install logstash-integration-aws plugin as follows. See also https://github.com/logstash-plugins/logstash-mixin-aws/issues/38
+```
+# Remove existing logstash aws plugins and install logstash-integration-aws to keep sdk dependency the same
```

Comment:
Thank you for adding this documentation. This is a significant step, so I think it will make sense to have this be a major version bump to `8.0.0`. Thoughts? @hardproblems @deepdatta

Reply:
major version bump sgtm

Reply:
@dlvenable updated. Will port this change to opensearch plugin as well

Reply:
https://github.com/opensearch-project/logstash-output-opensearch/pull/171 here's the logic ported over to opensearch

Reply:
Yes please 🙇